### DISCUSSION TITLE
Sort albums by artist

### DISCUSSION THREAD

**Discussion by SDoesburg** [music-assistant/support#2762] [Feature requests and ideas]
Perhaps I am overlooking this, but could you add an option to sort the albums view alphabetically by artist? I like being able to browse my collection in the album view but sorting the albums by title mixes them up and makes it hard to find albums where I don't remember the exact title but remember the artist and the cover art. I don't really mind how the albums are sorted within an artist, chronologically or alphabetically by title, but I think chronologically would work well.

I realise I could browse the artists through the artists view but I prefer to see a full list of albums instead of having to click into an artist and then an album.

**Comment by marcelveldt** [comment]
why not use search to type in the artist name ?

**Reply by SDoesburg** [reply]
That is certainly an option. I just like to browse the list of albums sorted by artist. It's been the predominant way I interact with my library since the early days of itunes and I was wondering If the option could be added. It's a small detail but I'd be nice to have.

**Comment by SDoesburg** [comment]
I just noticed that the sort by album artist in the album view is now available. Thank you so much!
